docs/develop/testing.md

```diff
@@ -78,7 +81,11 @@ Alternatively, you can run each of the containers manually as per [dev setup](se
 
```

Comment:
In the "Run the app" section above, can we rewrite it as below? 

```md
- Option 1 [Docker Compose]: See [Quick Start](../index.md#quick_start) for how to start the application stack using docker-compose.
- Option 2 [Dev Setup]: Run each container manually as per [dev setup](setup.md).
```

Reply:
I like it. Added